server/env.ts

```diff
@@ -766,6 +775,26 @@ export class Environment {
       );
     }
   }
+
+  /**
+   * Convert a string to a boolean. Supports the following:
+   *
+   * 0 = false
+   * 1 = true
+   * "true" = true
+   * "false" = false
+   * "" = undefined
+   *
+   * @param value The string to convert
+   * @returns A boolean or undefined
+   */
+  private toOptionalBoolean(value: string) {
```

Comment:
I'm not sure there is much value in this addition – better to use the existing `toBoolean` with a default of `false` than also have to handle `undefined`

Reply:
I did this to prevent the a possibly undefined env var from being sent across to the front end. While it doesn't make much difference to the end user, if they're not using OIDC and thus won't have the var set, I dont see the point in sending it across.

But it's up to you, I can just default to false if you like?

Reply:
> if they're not using OIDC and thus won't have the var set, I dont see the point in sending it across.

Seems reasonable, I'm convinced